src/Geta.Optimizely.Sitemaps/Services/NullUriAugmenterService.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using EPiServer.Core;
+using EPiServer.ServiceLocation;
+
+namespace Geta.Optimizely.Sitemaps.Services
+{
+    [ServiceConfiguration(typeof(IUriAugmenterService))]
+    public class NullUriAugmenterService : IUriAugmenterService
+    {
+        public IEnumerable<Uri> AugmentUri(IContent content, CurrentLanguageContent languageContentInfo, Uri fullUri)
```

Comment:
```suggestion
        public IEnumerable<Uri> GetAugmentUris(IContent content, CurrentLanguageContent languageContentInfo, Uri fullUri)
```

This method is a `query`, the name should reflect it.